AwsDscToolkit.psm1

```diff
@@ -922,6 +1004,10 @@ function Register-EC2Instance {
         [string]
         $AwsEncryptionKeyId,
 
+        [ValidateSet('Enable', 'Disable')]
+        [string]
```

Comment:
No. $null is only a valid value for the functions that this function passes to to indicate that the parameter was not supplied initially.